controller/planner_backlog.go

```diff
@@ -93,10 +93,11 @@ func generateBacklogExpression(ctx context.Context, db application.DB, spaceID u
 
 		// Get the list of work item types that derive of PlannerItem in the space
 		var expWits criteria.Expression
-		wits, err := appl.WorkItemTypes().ListPlannerItems(ctx, spaceID)
+		witsOrig, err := appl.WorkItemTypes().ListPlannerItems(ctx, spaceID)
 		if err != nil {
 			return errs.Wrap(err, "unable to fetch work item types that derive from planner item")
 		}
+		wits := stripBaseWorkItemTypes(witsOrig)
```

Comment:
> I agree it doesn't harm atm but if we have 1million of work item types to iterate over... then yes :)

We will only iterate over a handfull of WITs in the controller because the main filtering is done at repository level and the only WITs that are being returned are those associated with a space.